src/ogd/games/BLOOM/features/CountyPolicyChangeCount.py

```diff
@@ -0,0 +1,48 @@
+from typing import Any, Dict, List, Optional
+from ogd.core.generators.Generator import GeneratorParameters
+from ogd.core.generators.extractors.Feature import Feature
+from ogd.core.models.Event import Event
+from ogd.core.models.enums.ExtractionMode import ExtractionMode
+from ogd.core.models.FeatureData import FeatureData
+
+class CountyPolicyChangeCount(Feature):
```

Comment:
Resolved in 74c4744516db82ca51851a2e80cb59a67a42ce38

Reply:
Ack, made it per-county instead of per-count, dumb mistake, need to count what happened inside the specific county, but we may not be in there when alert occurs.